ext/web/02_event.js

```diff
@@ -787,7 +788,11 @@
 
     setCurrentTarget(eventImpl, tuple.item);
 
-    innerInvokeEventListeners(eventImpl, getListeners(tuple.item));
+    try {
+      innerInvokeEventListeners(eventImpl, getListeners(tuple.item));
+    } catch (error) {
+      __bootstrap.reportError.reportException(error);
```

Comment:
This is related to load order I guess? Annoying :/

Reply:
Yeah I don't really understand this bit. Why do we need to store whole `__bootstrap` namespace? `const reportError = window.__bootstrap.reportError;` should be enough.

Reply:
Please add a comment here explaining this logic